### PR TITLE
added endpoints ans db connection for the omplete printing and cuttin…

### PR DESCRIPTION
**Pull request by eineving**
…g phase

### CODE REVIEW

File: backend/src/main/java/WebApi.java [hunk 1]
```diff
@@ -78,6 +92,20 @@ private static void setupDigitalPartsInterface() {
         put(DIGITALPART + DIGITALPART_ID, (request, response) -> dpi.updateDigitalPart(gson.fromJson(request.body(), DigitalPart.class)), gson::toJson);
     }
 
+    private static void setupPhysicalInterface() {
```
Comment (chraker) [inline backend/src/main/java/WebApi.java]: MAJOR Remove this unused private "setupPhysicalInterface" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: backend/src/main/java/WebApi.java (hunk 1 above)
Comment (chraker) [inline backend/src/main/java/WebApi.java]: MAJOR Remove this unused private "setupPhysicalInterface" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: backend/src/main/java/WebApi.java (hunk 1 above)
Comment (chraker) [inline backend/src/main/java/WebApi.java]: MAJOR Remove this unused private "setupPhysicalInterface" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: backend/src/main/java/WebApi.java (hunk 1 above)
Comment (chraker) [inline backend/src/main/java/WebApi.java]: MAJOR Remove this unused private "setupPhysicalInterface" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: backend/src/main/java/WebApi.java (hunk 1 above)
Comment (chraker) [inline backend/src/main/java/WebApi.java]: MAJOR Remove this unused private "setupPhysicalInterface" method. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

**Comment by chraker** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by mlonn — APPROVED** [review]
Das is good

**Pull request merged**